static/src/components/setting/task_setting.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <div class="stats-config row">
+    <div class="col-md-12">
+      <div class="form-group row">
+        <div class="col-md-6">
+          <p> task scheduler </p>
+        </div>
+        <div class="col-md-6 pull-right">
+          <select
+            v-model="sched"
+            class="form-control input-sm"
+          >
+            <option
+              v-for="opt in getOptions()"
+              :key="opt.text"
+              :value="opt.value"
+            >
+              {{ opt.text }}
+            </option>
+          </select>
+        </div>
+      </div>
+      <div class="form-group row">
+        <div class="col-md-6">
+          <p> randomize order with same priority </p>
+        </div>
+        <div class="col-md-6 pull-right">
+          <label class="switch">
+            <input
+              v-model="random"
+              type="checkbox"
+            >
+            <span class="slider" />
```

Comment:
seems to also use style defined in another component?

Reply:
Maybe when we merge Kory PR you can do what she is doing to have a file with all  the css.

Reply:
scss can be used already